src/test/.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": ["airbnb-base", "plugin:jest/recommended"],
+  "env": {
+    "jest": true
+  },
+  "plugins": ["jest"],
+  "parser": "babel-eslint",
+  "rules": {
+    "indent": [2, 4],
+    "no-unused-vars": 0,
```

Comment:
well, I was focused on **repeat same behaviour** as at previous version of eslint. As for me - I would prefer to use as less **no-xxxx** settings as possible, but it will need more time to fix.